--- ppl-0.11.2/m4/ac_check_ciao.m4.ark	2012-01-10 20:38:20.796672253 +0100
+++ ppl-0.11.2/m4/ac_check_ciao.m4	2012-01-10 20:38:50.584299416 +0100
@@ -56,13 +56,14 @@ _CIAO_PRINT_INCLUDE_DIR_EOF
   AC_LANG_PUSH(C++)
   AC_CHECK_HEADER(ciao_prolog.h,
                   AC_MSG_CHECKING([for Ciao Prolog version x.y.z or later])
-                  AC_COMPILE_IFELSE(
-[#include <ciao_prolog.h>
+                  AC_COMPILE_IFELSE([AC_LANG_SOURCE([
+#include <ciao_prolog.h>
 
 int
 main() {
   return 0;
-}],
+}
+])],
                     AC_MSG_RESULT(yes),
                     ciao_prolog=""
                     AC_MSG_RESULT(no)
--- ppl-0.11.2/m4/ac_check_swi_prolog.m4.ark	2012-01-10 20:38:55.637236246 +0100
+++ ppl-0.11.2/m4/ac_check_swi_prolog.m4	2012-01-10 20:39:27.992831692 +0100
@@ -69,8 +69,8 @@ then
   # We require SWI-Prolog 5.6.0 or later.
   AC_CHECK_HEADER(SWI-Prolog.h,
                   AC_MSG_CHECKING([for SWI-Prolog version 5.6.0 or later])
-                  AC_COMPILE_IFELSE(
-[#include <SWI-Prolog.h>
+                  AC_COMPILE_IFELSE([AC_LANG_SOURCE([
+#include <SWI-Prolog.h>
 
 int
 main() {
@@ -80,7 +80,8 @@ main() {
 
   ;
   return 0;
-}],
+}
+])],
                     AC_MSG_RESULT(yes),
                     swi_prolog=""
                     AC_MSG_RESULT(no)
--- ppl-0.11.2/m4/ac_prog_javah.m4.ark	2012-01-10 20:31:27.063844174 +0100
+++ ppl-0.11.2/m4/ac_prog_javah.m4	2012-01-10 20:38:00.104930532 +0100
@@ -34,7 +34,9 @@ else
 fi
 
 if test x"`eval 'echo $ac_cv_path_JAVAH'`" != xno ; then
-  AC_PREPROC_IFELSE([#include <jni.h> ] ,,[
+  AC_PREPROC_IFELSE([AC_LANG_SOURCE([
+#include <jni.h>
+])],,[
     ac_save_CPPFLAGS="$CPPFLAGS"
 changequote(, )dnl
     ac_dir=`echo $ac_cv_path_JAVAH | sed 's,\(.*\)/[^/]*/[^/]*$,\1/include,'`
@@ -43,7 +45,9 @@ changequote([, ])dnl
     JNIFLAGS="-I$ac_dir -I$ac_dir/$ac_machdep"
     CPPFLAGS="$ac_save_CPPFLAGS $JNIFLAGS"
     AC_SUBST(JNIFLAGS)
-    AC_PREPROC_IFELSE([#include <jni.h>],
+    AC_PREPROC_IFELSE([AC_LANG_SOURCE([
+#include <jni.h>
+])],
                CPPFLAGS="$ac_save_CPPFLAGS",
                AC_MSG_WARN([unable to include <jni.h>])
 	       JAVAH=no)
